NEWS

```diff
@@ -18,6 +18,20 @@ CHANGES WITH 253 in spe:
           For more details, see:
           https://lists.freedesktop.org/archives/systemd-devel/2022-September/048352.html
 
+        * We intend to change behaviour w.r.t. user units and sandboxing
+          options, so that they work in the same way as system units by default.
+          This requires implicitly enabling user namespaces (PrivateUsers=yes)
+          when a sandboxing option is enabled in a user unit. The drawback is
+          that system users will no longer be visible (and apper as 'nobody') to
```

Comment:
typo

Reply:
i think this should could use clarification, it's "objects owned by other system users will now appear as owned by the user "nobody"